msos/StackWalker.cs

```diff
@@ -85,6 +85,18 @@ protected override void ExtractWaitForSingleObjectInformation(UnifiedStackFrame
                 AddSingleWaitInformation(frame, handle);
             }
         }
+
+        protected override void ExtractSleepObjectInformation(UnifiedStackFrame frame)
+        {
+            var parameters = GetParameters(frame, SLEEP_FUNCTION_OBJECT_PARAM_COUNT);
+            if (parameters.Count > 0)
+            {
+                var handle = ConvertToAddress(parameters[0]);
+                
+                UnifiedHandle unifiedHandle = new UnifiedHandle(handle, nameof(UnifiedBlockingReason.ThreadSleep), null);
```

Comment:
Not sure what this is. When you call `Sleep`, there is no handle. There is just a timeout. Did you test this?